addons/mail/static/src/owl/components/file_uploader/file_uploader.js

```diff
@@ -188,6 +190,7 @@ FileUploader.props = {
     },
     fileUploadId: {
         type: String,
+        optional: true,
     },
```

Comment:
Don't see any template with `fileUploadId` as prop... Maybe drop it and rely entirely on `_.uniqueId()`?

Reply:
I was just leaving the possibility to set it manually if needed, you think it's not useful ?

Reply:
Would remove it